cmd/provisioner-localpv/app/helper_hostpath.go

```diff
@@ -165,9 +169,10 @@ func (p *Provisioner) launchPod(config podConfig) (*corev1.Pod, error) {
 	privileged := true
 
 	helperPod, err := pod.NewBuilder().
-		WithName(config.podName + "-" + config.pOpts.name).
+		WithName(config.podName+"-"+config.pOpts.name).
```

Comment:
```suggestion
		WithName(config.podName + "-" + config.pOpts.name).
```

formatting change, its better to have spaces.

Reply:
👍